RevenueCat/Scripts/SubscriptionInfo.cs

```diff
@@ -0,0 +1,88 @@
+using UnityEngine;
+using System;
+using RevenueCat.SimpleJSON;
+using static RevenueCat.Utilities;
+
+
+public partial class Purchases
+{
+    
+    /// <summary>
+    /// The SubscriptionInfo object gives you access to all of the information about the status of a subscription.
+    /// </summary>
+    public class SubscriptionInfo
+    {
+        public readonly string ProductIdentifier;
+        public readonly DateTime PurchaseDate;
+        public readonly DateTime? OriginalPurchaseDate;
+        public readonly DateTime? ExpiresDate;
+        public readonly string Store;
+        public readonly DateTime? UnsubscribeDetectedAt;
+        public readonly bool IsSandbox;
+        public readonly DateTime? BillingIssuesDetectedAt;
+        public readonly DateTime? GracePeriodExpiresDate;
+        public readonly string OwnershipType;
+        public readonly string PeriodType;
+        public readonly DateTime? RefundedAt;
+        public readonly string? StoreTransactionId;
+        public readonly bool IsActive;
+        public readonly bool WillRenew;
+
+        public SubscriptionInfo(JSONNode response)
+        {
+            ProductIdentifier = response["productIdentifier"];
+            var purchaseDateTime = FromISO8601(response["purchaseDate"]);
+            if (purchaseDateTime == null) {
+                Debug.LogError("Purchase date is null or has an invalid format. Defaulting to 1970-01-01.");
+            }
+            PurchaseDate = purchaseDateTime ?? new DateTime(1970, 1, 1);
+            OriginalPurchaseDate = FromResponseISO8601String(response, "originalPurchaseDate");
+            ExpiresDate = FromResponseISO8601String(response, "expiresDate");
+            Store = response["store"];
+            UnsubscribeDetectedAt = FromResponseISO8601String(response, "unsubscribeDetectedAt");
+            IsSandbox = response["isSandbox"].AsBool;
+            BillingIssuesDetectedAt = FromResponseISO8601String(response, "billingIssueDetectedAt");
+            GracePeriodExpiresDate = FromResponseISO8601String(response, "gracePeriodExpiresDate");
+            OwnershipType = response["ownershipType"];
+            PeriodType = response["periodType"];
```

Comment:
Should we have enums for `OwnershipType` and `PeriodType`?

Reply:
Good question... So `PeriodType` is used in `EntitlementInfo` and doesn't have it. That + us deciding not to use enums made me think to avoid it... Not an expert in C# to know what would be the best alternative though 😓. Wdyt?

Reply:
if `EntitlementInfo` doesn't have it, then I think we are good!